libs/examples/src/empty-state/empty-state-multiple-items-example/empty-state-multiple-items-example.html

```diff
@@ -4,7 +4,7 @@
       <img src="/assets/cta-noagent.svg" alt="No agent" />
     </dt-empty-state-item-img>
 
-    <dt-empty-state-item-title>
+    <dt-empty-state-item-title aria-level="2">
```

Comment:
Do we also have to add `role="heading"` then?

I'm not sure if we should add the `aria-level` attribute here as we don't know where the code of this example is copied to. I think it can happen, that the code only gets copy-pasted without this value being changed (in case it needs to be changed because of the page hierarchy). Should we add a comment that this value needs to be changed if the empty-state-title is not a 2nd level headline?

Reply:
The `role=heading`is set by the empty state title component. I think we should include it, otherwise the a11y tests for this component fails, and it is not clear that a level should be set at all.